detect_secrets/plugins/keyword.py

```diff
@@ -275,24 +313,26 @@ def secret_generator(self, string, filetype):
 
 def probably_false_positive(lowered_secret, filetype):
     if (
-        'fake' in lowered_secret
-        or 'forgot' in lowered_secret
-        or lowered_secret in FALSE_POSITIVES
-        or (
-            filetype == FileType.JAVASCRIPT
-            and (
-                lowered_secret.startswith('this.')
-                or lowered_secret.startswith('fs.read')
-                or lowered_secret.startswith('options.')
-                or lowered_secret == 'new'
+        any(
+            false_positive in lowered_secret
+            for false_positive in (
+                '/etc/',
+                'fake',
+                'forgot',
             )
+        ) or lowered_secret in FALSE_POSITIVES
+        or lowered_secret.count('/') >= 3
```

Comment:
Why?

Reply:
Good catch, added comment in https://github.com/Yelp/detect-secrets/pull/229/commits/5de43e278169ba8f040f6cf24b7b41e134d265fc